packages/odyssey-react-mui/src/OdysseyProvider.tsx

```diff
@@ -60,7 +64,8 @@ const OdysseyProvider = <SupportedLanguages extends string>({
       shadowRootElement={shadowRootElement || shadowDomElement}
       themeOverride={themeOverride}
     >
-      <ScopedCssBaseline>
+      {/* This component creates a div; for flexibility of layout of children, make it inherit its parent's height */}
+      <ScopedCssBaseline sx={scopedCssBaselineStyles}>
```

Comment:
I've discussed this with @KevinGhadyani-Okta and the need to allow this to respect its parent's wishes

Reply:
👍